.github/workflows/linux.yml

```diff
@@ -23,14 +23,14 @@ on:
     types: [checks_requested]
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.number && format('pr{0}', github.event.number) || github.run_id }}
-  cancel-in-progress: ${{ github.event_name == 'merge_group' }}
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
Just curious, did this affect anything?

Reply:
`cancel-in-progress` part? Yes. 

`group`? Not necessarily. I updated it just to be aligned with the [official docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs#example-using-a-fallback-value). I can revert it if you would prefer that.

Reply:
Yeah I meant `group`. Makes sense.